README.md

```diff
@@ -29,15 +29,22 @@ To run, please import these credentials into the app:
 ```bash
 export AWS_ACCESS_KEY_ID=<KEY_ID>
 export AWS_SECRET_ACCESS_KEY=<SECRET_KEY>
+export API_BUCKET_NAME=sdp-dev-tech-audit-tool-api
+export API_SECRET_NAME=sdp-dev-tech-audit-tool-api/secrets
```

Comment:
> Since you have defined a variable for the bucket name you can use it in the next line:
> 
> export API_SECRET_NAME=$API_BUCKET_NAME/secrets

Fixed